cli/eslint.config.js

```diff
@@ -12,7 +12,7 @@ export default antfu({
         'no-console': [
             'error',
             {
-                allow: ['info', 'warn', 'error', 'table'],
+                allow: ['info', 'warn', 'error', 'table', 'trace'],
```

Comment:
Q: do we need this `trace` added here?

Reply:
Not anymore, but it's fine to keep it as well 🙂